powerauth-rest-client-spring/src/main/java/com/wultra/security/powerauth/rest/client/PowerAuthRestClient.java

```diff
@@ -1043,6 +1050,46 @@ public RemoveApplicationRolesResponse removeApplicationRoles(Long applicationId,
         return removeApplicationRoles(request);
     }
 
+    @Override
+    public OperationDetailResponse createOperation(OperationCreateRequest request) throws PowerAuthClientException {
+        return callV3RestApi("/operation/create", request, OperationDetailResponse.class);
+    }
+
+    @Override
+    public OperationDetailResponse operationDetail(OperationDetailRequest request) throws PowerAuthClientException {
```

Comment:
Can we add a variant of this method with operation ID as a parameter?

Reply:
No.